src/contractor/contractor.cpp

```diff
@@ -193,10 +193,14 @@ int Contractor::Run()
 
     TIMER_STOP(preparing);
 
+    const auto nodes_per_second =
```

Comment:
To be clear here this works with an additional `std::fixed`, too - no reason I chose the `uint64_t` over the format specifiers other that I don't like streams much.